### DISCUSSION TITLE
How to configure object storage as a backend?

### DISCUSSION THREAD

**Discussion by woile** [scaleway/terraform-provider-scaleway#2392] [Q&A] [answered]
Hello, I'm trying to configure an Object Storage as a backend for terraform but I cannot find any docs.
Is there some resource I can read?

Regards

**Comment by Codelax** [comment, marked as answer]
Hi, this is quickly mentioned in our [provider's documentation](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs#store-terraform-state-on-scaleway-s3-compatible-object-storage).

**Reply by woile** [reply]
Thanks, that's fantastic. Google was not giving any result. Only the example about using postgres as a backend.
